design-toolkit/components/src/tokens/generated/tokens.ts

```diff
@@ -0,0 +1,133 @@
+export const colorsHighlightPrimary05: RGBAColor = [10, 61, 48, 1];
+export const colorsHighlightSecondary01: RGBAColor = [201, 246, 253, 1];
+export const colorsHighlightSecondary02: RGBAColor = [126, 233, 249, 1];
+export const colorsHighlightSecondary03: RGBAColor = [40, 220, 245, 1];
+export const colorsHighlightSecondary04: RGBAColor = [24, 132, 147, 1];
+export const colorsHighlightSecondary05: RGBAColor = [10, 55, 61, 1];
+export const colorsHighlightTertiary01: RGBAColor = [249, 197, 226, 1];
+export const colorsHighlightTertiary02: RGBAColor = [241, 116, 185, 1];
+export const colorsHighlightTertiary03: RGBAColor = [232, 23, 138, 1];
+export const colorsHighlightTertiary04: RGBAColor = [138, 14, 83, 1];
+export const colorsHighlightTertiary05: RGBAColor = [58, 6, 35, 1];
+export const colorsUtilityInfo01: RGBAColor = [230, 230, 230, 1];
+export const colorsUtilityInfo02: RGBAColor = [137, 137, 137, 1];
+export const colorsUtilityInfo03: RGBAColor = [104, 104, 104, 1];
+export const colorsUtilityInfo04: RGBAColor = [66, 66, 68, 1];
+export const colorsUtilityInfo05: RGBAColor = [34, 34, 34, 1];
+export const colorsUtilityAdvisory01: RGBAColor = [224, 237, 255, 1];
+export const colorsUtilityAdvisory02: RGBAColor = [161, 202, 255, 1];
+export const colorsUtilityAdvisory03: RGBAColor = [98, 166, 255, 1];
+export const colorsUtilityAdvisory04: RGBAColor = [58, 100, 153, 1];
+export const colorsUtilityAdvisory05: RGBAColor = [20, 33, 51, 1];
+export const colorsUtilityNormal01: RGBAColor = [214, 246, 229, 1];
+export const colorsUtilityNormal02: RGBAColor = [130, 228, 178, 1];
+export const colorsUtilityNormal03: RGBAColor = [48, 210, 126, 1];
+export const colorsUtilityNormal04: RGBAColor = [28, 126, 76, 1];
+export const colorsUtilityNormal05: RGBAColor = [10, 42, 25, 1];
+export const colorsUtilitySerious01: RGBAColor = [254, 237, 204, 1];
+export const colorsUtilitySerious02: RGBAColor = [253, 200, 101, 1];
+export const colorsUtilitySerious03: RGBAColor = [252, 164, 0, 1];
+export const colorsUtilitySerious04: RGBAColor = [151, 98, 1, 1];
+export const colorsUtilitySerious05: RGBAColor = [50, 33, 0, 1];
+export const colorsUtilityCritical01: RGBAColor = [246, 211, 210, 1];
+export const colorsUtilityCritical02: RGBAColor = [229, 123, 119, 1];
+export const colorsUtilityCritical03: RGBAColor = [212, 35, 29, 1];
+export const colorsUtilityCritical04: RGBAColor = [127, 21, 17, 1];
+export const colorsUtilityCritical05: RGBAColor = [42, 7, 6, 1];
+export const colorsClassificationUnclass: RGBAColor = [0, 122, 51, 1];
+export const colorsClassificationCui: RGBAColor = [80, 43, 133, 1];
+export const colorsClassificationConfidential: RGBAColor = [0, 51, 160, 1];
+export const colorsClassificationSecret: RGBAColor = [200, 16, 46, 1];
+export const colorsClassificationTopSecret: RGBAColor = [255, 140, 0, 1];
+export const colorsClassificationTsSci: RGBAColor = [252, 232, 58, 1];
+export const typographyHeaderXxlSize = '32px';
```

Comment:
These should be numbers instead of strings

Reply:
updated